### PR TITLE
mgr/dashboard: Improve descriptions in some parts of the dashboard

### PR DESCRIPTION
**Pull request by nizamial09**
Improves the description in some parts of the dashboard code.

restoretrash
movetotrash
purge
role_description

Fixes: https://tracker.ceph.com/issues/49829
Fixes: https://tracker.ceph.com/issues/49969
Signed-off-by: Nizamudeen A <nia@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by avanthakkar — APPROVED** [review]
LGTM!

**Review by aaSharma14 — APPROVED** [review]

**Review by alfonsomthd — APPROVED** [review]

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-purge-modal/rbd-trash-purge-modal.component.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
           novalidate>
       <div class="modal-body">
         <p>
-          <ng-container i18n>To purge, select one or</ng-container>&nbsp;
+          <ng-container i18n>To purge, select</ng-container>&nbsp;
```
[the remark this answers is not in the document]
Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-purge-modal/rbd-trash-purge-modal.component.html:13]: Yeah, makes sense to me, thanks. I'll make the changes and will notify you once it is updated. @alfonsomthd

[the remark this answers is not in the document]
Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-purge-modal/rbd-trash-purge-modal.component.html]: Done.

**Review by callithea — APPROVED** [review]
Lgtm

**Comment by nizamial09** [comment]
jenkins test dashboard

**Pull request merged**